camel-core/src/main/java/org/apache/camel/component/xslt/XsltEndpoint.java

```diff
@@ -71,6 +72,8 @@
     @UriParam
     private boolean saxon;
     @UriParam(label = "advanced")
+    private List<Object> saxonExtensionFunctions;
```

Comment:
Is there any out-of-the box support in camel ? or I need to define saxonExtensionFunctions as String and then split/lookup values in the component ?